src/Analysis/Ast/Impl/Specializations/BuiltinsSpecializations.cs

```diff
@@ -91,6 +92,23 @@ public static IMember List(IPythonInterpreter interpreter, IPythonFunctionOverlo
             return args.Count > 0 && args[0] is PythonCollection c ? c.Contents.FirstOrDefault() : null;
         }
 
+        public static IMember Super(IPythonModule declaringModule, IPythonFunctionOverload overload, IArgumentSet argSet, IndexSpan indexSpan) {
+            foreach (var s in argSet.Eval.CurrentScope.EnumerateTowardsGlobal) {
+                if (s.Node is ClassDefinition) {
+                    var classType = s.Variables["__class__"].GetPythonType<IPythonClassType>();
```

Comment:
`s.Node is ClassDefinition cd`
`s.Variables[cd.Name]?.Value as IPythonClassType`

Reply:
s.Variables[cd.Name]?.Value as IPythonClassType wasn't working for me. I'll take another look

Reply:
Yeah b/c variable is declared in outer scope. I.e. `s.OuterScope.Variables[cd.Name]?.Value as IPythonClassType`. But `__class__` is fine too.